tools/cldr-apps/js/src/esm/cldrVettingParticipation.js

```diff
@@ -112,9 +115,11 @@ async function downloadVettingParticipation(opts) {
       "Code",
       "Level",
       "Votes",
-      "CldrCovCount",
-      "ProgressVote",
-      "ProgressCount",
+      "Cldr Coverage Count",
```

Comment:
In that case this should be changed to just Cov Count, and code chanege to be the org's, not CLDR 's. (Whole point of this sheet now is being from the org's perspective)

Also, the votes should be restricted to the org's coverage

Reply:
That is, if a vetter votes for a modern path, but the org's coverage is moderate, then that vote isn't included in Votes.

Reply:
@macchiati I don't think it works that way now, so I'll take "Should" as meaning "please change it to…"

In any event, merged this because it makes two improvements as already requested, and I'll add your request to the ticket.